From 7d9101c3cc5729428eb2b7ae9ce1246bac28fa5e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Maxime=20M=C3=A9r=C3=A9?= <maxime.mere@foss.st.com>
Date: Fri, 25 Oct 2024 14:36:27 +0200
Subject: [PATCH 1338/1695] dts: stm32: add RNG support for STM32MP21
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Update RNG nodes to support STM32MP21

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git e0098f3259450f649bce8ba8ebe5f44efa86fb9e dts: stm32: add RNG support for STM32MP21]
Signed-off-by: Maxime Méré <maxime.mere@foss.st.com>
Change-Id: I27bc8e4394876150126886897a928a61ffb858d1
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/414255
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 core/arch/arm/dts/stm32mp211.dtsi  | 2 +-
 core/arch/arm/dts/stm32mp21xc.dtsi | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp211.dtsi b/core/arch/arm/dts/stm32mp211.dtsi
index 8f90c8a77..a09dc3942 100644
--- a/core/arch/arm/dts/stm32mp211.dtsi
+++ b/core/arch/arm/dts/stm32mp211.dtsi
@@ -2248,7 +2248,7 @@
 			};
 
 			rng2: rng@42020000 {
-				compatible = "st,stm32mp25-rng";
+				compatible = "st,stm32mp21-rng";
 				reg = <0x42020000 0x400>;
 				clocks = <&clk_rcbsec>, <&rcc CK_BUS_RNG2>;
 				clock-names = "rng_clk", "rng_hclk";
diff --git a/core/arch/arm/dts/stm32mp21xc.dtsi b/core/arch/arm/dts/stm32mp21xc.dtsi
index 7ddf42d4b..38068a46e 100644
--- a/core/arch/arm/dts/stm32mp21xc.dtsi
+++ b/core/arch/arm/dts/stm32mp21xc.dtsi
@@ -24,7 +24,7 @@
 	};
 
 	rng1: rng@42030800 {
-		compatible = "st,stm32mp25-rng";
+		compatible = "st,stm32mp21-rng";
 		reg = <0x42030800 0x400>;
 		clocks = <&clk_rcbsec>, <&rcc CK_BUS_RNG1>;
 		clock-names = "rng_clk", "rng_hclk";
-- 
2.47.2

